test/unit/deploy/deploy_info_test.rb

```diff
@@ -2,7 +2,7 @@
 
 require 'test_helper'
 
-class BasicInfoTest < ActiveSupport::TestCase
+class DeployInfoTest < ActiveSupport::TestCase
```

Comment:
Should class be `InfoTest` since the module part `Deploy` is in the path? 🤔 
```
module Deploy
  class Info ->  deploy/info_test.rb

  class InvalidInfo -> deploy/invalid_info_test.rb
end
```

Reply:
I guess ideally, it would go to `unit/lib/system/deploy_test.rb` to match `lib/system/deploy.rb`

But I thought this wasn't important... I mean, not worth spending time trying to figure out the best name :)
I just eliminated BasicInfo to avoid confusion.